packages/@aws-cdk/aws-backup/README.md

```diff
@@ -44,6 +44,9 @@ created for the selection. The `BackupSelection` implements `IGrantable`.
 
 To add rules to a plan, use `addRule()`:
 
+And use the Volume Shadow Copy Service via the windowsVss parameter, default value is false. 
+To use this, you can provide windowsVss as a parameter to new backup.BackupPlanRule.
+
```

Comment:
This should be located in a separate section; probably after the current line 77. It should also provide a code example which can replace the unclear second sentence ("To use this, ..."). It also needs to have a more detailed explanation of what VSS actually is and why someone would use it. Feel free to pull the initial documentation directly from the Developer Guide. In particular, this should mention that there are steps that need to be taken on the instances themselves